docs/source/markdown/podman-create.1.md

```diff
@@ -80,7 +80,7 @@ If the host uses cgroups v1, the default is set to **host**.  On cgroups v2 the
 Determines whether the container will create CGroups.
 Valid values are *enabled*, *disabled*, *no-conmon*, which the default being *enabled*.
 The *disabled* option will force the container to not create CGroups, and thus conflicts with CGroup options (**--cgroupns** and **--cgroup-parent**).
-The *no-conmon* option disables a new CGroup only for the conmon process.
+The *no-conmon* option disables a new CGroup only for the conmon process.  The *split-current* option reuses the current cgroup for both the conmon and the container payload.
```

Comment:
Can we just call this current?  I don't see where split- adds any additional information to the user.

Reply:
we may need `current` in future to really mean `current` and use the cgroup where Podman is running instead of creating two sub-cgroups

Reply:
`reuse-current` or perhaps just `reuse` then?